modules/extensions-support/src/main/java/org/mule/module/extension/internal/ExtensionProperties.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal;
+
+import static org.mule.api.config.MuleProperties.PROPERTY_PREFIX;
+
+/**
+ * Constants for the Extensions Framework
+ *
+ * @since 4.0
+ */
+public final class ExtensionProperties
```

Comment:
why? I don't want anybody with the brillant idea of extending this class. What's the harm?